doc/releases/release-notes-3.7.rst

```diff
@@ -18,8 +18,9 @@ Major enhancements with this release include:
 * A long-awaited HTTP server library, and associated service API, allow to easily implement HTTP/1.1
   and HTTP/2 servers in Zephyr. Resources can be registered statically or at runtime, and WebSocket
   support is included.
-* POSIX support has been extended, with all the Option Requirements of POSIX Subprofiling Option
-  Groups now being supported for PSE51, PSE52, and PSE53 profiles.
+* POSIX support has been extended, with most Options of the IEEE 1003-2017 System Interfaces (
+  Base Definitions) receiving support, as well as most Options and Option groups required for
```

Comment:
nit: not sure but i'm concerned that the line breaking at `(` is probably ending up rendering with a whitespace after it, so it's prob better to keep `(Base` together?

Reply:
Fixed!d